requirements_all_ds.txt

```
@@ -1,34 +1,34 @@
-google-api-python-client==1.5.1
+google-api-python-client==1.7.11
 gspread==3.1.0
-impyla==0.10.0
-influxdb==2.7.1
+impyla==0.16.0
+influxdb==5.2.3
 mysqlclient==1.3.14
-oauth2client==3.0.0
-pyhive==0.5.1
-pymongo[tls,srv]==3.6.1
-vertica-python==0.8.0
-td-client==0.8.0
-pymssql==2.1.3
-dql==0.5.24
-dynamo3==0.4.7
-boto3==1.9.115
-botocore>=1.12.239,<1.13.0
+oauth2client==4.1.3
+pyhive==0.6.1
+pymongo[tls,srv]==3.9.0
+vertica-python==0.9.5
+td-client==1.0.0
+pymssql==2.1.4
+dql==0.5.26
+dynamo3==0.4.10
+boto3>=1.9.241,<1.10.0
+botocore>=1.12.241,<1.13.0
 sasl>=0.1.3
 thrift>=0.8.0
 thrift_sasl>=0.1.0
-cassandra-driver==3.11.0
+cassandra-driver==3.19.0
 memsql==3.0.0
-atsd_client==2.0.12
-simple_salesforce==0.72.2
+atsd_client==3.0.5
+simple_salesforce==0.74.3
 PyAthena>=1.5.0
-pymapd==0.7.1
+pymapd==0.16.0
 qds-sdk>=1.9.6
 ibm-db>=2.0.9
-pydruid==0.4
-requests_aws_sign==0.1.4
-snowflake_connector_python==1.8.6
+pydruid==0.5.5
+requests_aws_sign==0.1.5
+snowflake_connector_python==2.0.0
 phoenixdb==0.7
 # certifi is needed to support MongoDB and SSL:
-certifi
-pydgraph==1.2.0
-azure-kusto-data==0.0.32
+certifi==2019.9.11
```

```suggestion
certifi>=2019.9.11
```
This is the kind of dependencies that is probably safe to leave unversioned (added a lower boundary to appease linters).